Makefile

```diff
@@ -7,6 +7,12 @@ AWS_PROFILE := default
 
 PYTEST_OPTS := ''
 
+PRODUCTION_PYTHON := 3.6
```

Comment:
nit: seems weird to mention production python in the OS repo. Should we just check for versions we don't test against in CI?

Reply:
agreed -- I'll switch to PYTHON_MIN_VERSION